sale_order_product_recommendation/models/sale_order_line.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
```

Comment:
Please don't change the name of the file. It's OK to host in the "parent" model adaptations of the children one, so that if in the future, something at `sale.order` needs to be added, you don't have to rename the file again.